projects/cds/src/merchandising/facade/cds-merchandising-product.service.ts

```diff
@@ -23,12 +29,16 @@ export class CdsMerchandisingProductService {
     return combineLatest([
       this.baseSiteService.getActive(),
       this.languageService.getActive(),
+      this.routingService
+        .getRouterState()
+        .pipe(switchMap(state => of(state.state.params['productCode']))),
```

Comment:
this can be flatten a bit by doing `map(state => state.state.params['productCode'])`